nixos/modules/services/networking/ejabberd.nix

```diff
@@ -149,13 +138,18 @@ in {
           for src in ${dumps}; do
             find "$src" -type f | while read dump; do
               echo "Loading configuration dump at $dump"
-              chpst -u "${cfg.user}:${cfg.group}" ${ectl} load "$dump"
+              ${ectl} load "$dump"
             done
           done
         fi
       '';
     };
 
+    systemd.tmpfiles.rules = [
+      "d ${cfg.logsDir} 0750 ${cfg.user} ${cfg.group} -"
+      "d ${cfg.spoolDir} 0750 ${cfg.user} ${cfg.group} -"
```

Comment:
Please escape `${cfg.logsDir}` and `${cfg.spoolDir}` with single quotes.

Reply:
Like [this](https://github.com/NixOS/nixpkgs/pull/59731/commits/f9406ddc080209ef38f9bcaf018c25461bbdebae#diff-f4ad427ecde914245907dd54f8d02c4dR149)?

Reply:
Yes. Though I now notice you're missing the final parameter. Please add one more `-` to each entry.

Reply:
I am? Neither the examples in the manual, e.g. 
```
# /usr/lib/tmpfiles.d/abrt.conf
d /var/tmp/abrt 0755 abrt abrt -
```
nor the examples in nixpkgs, e.g. [iwd](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/networking/iwd.nix#L28) have it.

Reply:
Ah you're totally right. Even the [manual](https://www.freedesktop.org/software/systemd/man/tmpfiles.d.html) has an example like that. Silly me. Sorry :smile: